src/pal/src/thread/process.cpp

```diff
@@ -3945,6 +3964,19 @@ PROCGetProcessStatus(
     return palError;
 }
 
+bool GetApplicationContainerFolder(PathCharString& buffer, const char *applicationGroupId, int applicationGroupIdLength)
```

Comment:
is this worth wrapping in #ifdef __APPLE__ ?

Reply:
I think it makes sense to wrap this too. Its very apple specific. Maybe even change the name to something apple specific ?